json-schemas/src/client-events-api-requests.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably API Requests Client Event",
+  "type": "object",
+  "description": "A message sent on the clientEvents:apiRequests metachannel describing a sampled client api request.",
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "The HTTP Host header of the API request."
+    },
+    "requestId": {
+      "type": "string",
+      "description": "The unique ID of the request that Ably can use to correlate a request event with internal logs, if necessary."
+    },
+    "site": {
+      "type": "string",
+      "description": "The site the request was processed in."
+    },
+    "headers": {
+      "type": "object",
+      "description": "The HTTP headers sent with the request. Keys should be lower case."
```

Comment:
"should" is too strong, it's not a requirement, it's just the convention we use:

```
Header names are lower case (e.g. 'accept', 'content-type', 'user-agent', etc.)
```